app/scripts/templates/settings/change_password.mustache

```diff
@@ -23,7 +23,7 @@
           {{#t}}Show{{/t}}
         </label>
 
-        <div class="input-help links"><a href="/reset_password" class="reset-password">{{#t}}Forgot password?{{/t}}</a></div>
+        <div class="input-help-forgot-pw links"><a href="/reset_password" class="reset-password">{{#t}}Forgot password?{{/t}}</a></div>
```

Comment:
do we want this class to be set to `input-help input-help-forgot-pw` to not affect any style changes from [_input-row.scss](https://github.com/mozilla/fxa-content-server/blob/master/app/styles/modules/_input-row.scss) or [_password_row.scss](https://github.com/mozilla/fxa-content-server/blob/master/app/styles/modules/_password-row.scss)?